src/sqlfluff/rules/L019.py

```diff
@@ -77,6 +76,16 @@ def _last_code_seg(raw_stack):
                 return segment
         return None
 
+    @staticmethod
+    def _follows_seg(
```

Comment:
Not loving the name here. I'm not a Python developer but is there a better name? Like `_get_following_seg` or `get_next_seg`?

Reply:
👍  I'll change it to `_get_following_seg`.